doc/UNT0019.md

```diff
@@ -0,0 +1,39 @@
+# UNT0019 Unnecessary indirection call for GameObject.gameObject and Transform.transform
+
+Unity `GameObject` has a property `gameObject` that returns `this` and same for
```

Comment:
Indeed `GameObject` has a property `gameObject`returning `this`. But for Transform it is a bit different.
Transform inherits from `Component` and a `Component` exposes `gameObject` and `transform` properties. Of course Transform.transform will return `this`. In the end we are looking for the same unnecessary indirection. Do you plan to support this case in another PR ?

Reply:
Yes, I just removed the reference to Transform and will do this in another PR.

Reply:
Perfect. Thanks.